app/models/article.rb

```diff
@@ -1,4 +1,10 @@
 class Article < ApplicationRecord
+  def to_meta_tags
+    {
+      title: title,
+      description: text,
```

Comment:
this would display the full article. let's only show the first 300 characters of the article.